reference/shipping.v2.yml

```
@@ -1400,6 +1434,25 @@ paths:
             "shipping_cost": 8
         }
         ```
+        
+        ### Channels
+
+                Example request body: 
+
+                ```json
+                {
+                  "name": "Method associated to channels 1,3",
+                  "type": "peritem",
+                  "settings": {
+                    "rate": 5
+                  },
+                  channels: [1, 3]
+                }
+                ```
+
+                | Property | Type | Description |
+                |:---------|:-----|:------------|
```

```suggestion
                | - | - | - |
```

I think we can replace the above with the simplified definition, a little easier to maintain, since this updated definition doesn't have any relation to the header row's column name length. As seen in `Range Object – Properties` example.

---

Thanks for the suggestion Brett, yeah exactly. I've updated all the instances since we're here. 🍻

---

Lovely sir, thanks!